@stellar/anchor-tests/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
this is a pattern we used in Freighter. We create a base tsconfig in the root, and subrepo's can extend that to prevent duplication of configs.

Up to you if you want to keep this